Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+import BraveShields
+import BraveVPN
+import Data
+
+struct SubmitReportView: View {
+  @Environment(\.dismiss) private var dismiss: DismissAction
+  let url: URL
+  let isPrivateBrowsing: Bool
+  
+  @State private var additionalDetails = ""
+  @State private var contactDetails = ""
+  @State private var isSubmittingReport = false
+  @State private var isSubmitted = false
+  
+  private var scrollContent: some View {
+    ScrollView {
+      // When we drop iOS 16, we can use NavigationStack with .navigationDestination
+      // For now we use a hack available to iOS 16
+      NavigationLink(isActive: $isSubmitted) {
+        SubmitReportSuccessView()
+          .navigationBarHidden(true)
+      } label: {
+        EmptyView()
+      }
```

Comment:
Perhaps I don't understand. But If I put this in the background of the scroll view, it will be behind the content. Do you intend me to hide the content or something? I'm not sure I like the effect this causes. 

Please let me know if I'm misunderstanding your suggestion.

Reply:
No, to clarify:
- Right now it is part of the `VStack` and technically takes part in the layout cycle (though its an `EmptyView` so it'd technically just say "my size is 0"). 
- A `NavigationLink` that takes an `isActive` argument and an `EmptyView` label doesn't need to be part of the visible view hierarchy since it is not something the user will interact with. 
- This will not change the behaviour of how the `SubmitReportSuccessView` is presented
- This change doesn't actually make any difference to the user, it is just semantically nicer for us, it matches more of the `navigationDestination(for:_:)` modifier that we'd replace it with since its only purpose is to actually trigger the navigation

Reply:
Im so sorry, I must have been distracted and didn't read your comment properly. This all makes sense to me and I made the appropriate adjustments. I tested it and it works just fine.